csuntechlab/iSTART
# STR-44 -- FE: Add yarn command for automating the commit process

# Description
Added a yarn commit command to help automate the commit process; runs prod, stages changes, and asks for commit.
  
# Testing
1. Create a test branch based of STR-44.
2. Make any changes to a file.
3. Run `git status` to ensure there are changes that need to be stage.
4. Run `yarn commit` or `yarn run commit` and enter a commit message, _**Do not include quotes**_, the command does it for you.
5. Run `git log` and ensure the commit you made was added.
